content/source/docs/cloud/run/cli.html.md

```diff
@@ -11,7 +11,7 @@ page_title: "CLI-driven Runs - Runs - Terraform Cloud and Terraform Enterprise"
 
 # The CLI-driven Run Workflow
 
-> For a hands-on tutorial, try the [Authenticate the CLI with Terraform Cloud](https://learn.hashicorp.com/terraform/tfc/tfc_login?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) guide on HashiCorp Learn.
+> **Hands-on:** Try the [Authenticate the CLI with Terraform Cloud](https://learn.hashicorp.com/tutorials/terraform/cloud-login?in=terraform/0-13&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial on HashiCorp Learn.
```

Comment:
Should we look into changing this tutorial's name?

We're not really authenticating the CLI **with** TFC, but rather authenticating the CLI **to** TFC, right?

Reply:
I'll leave that question for another day.